qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -80,10 +80,9 @@ class InterleavedRB(StandardRB):
 
             int_expdata2 = int_exp2.run(backend=backend).block_for_results()
             int_results2 = int_expdata2.analysis_results()
-
             display(int_expdata2.figure(0))
-            for result in int_results2:
-                print(result)
+
+            print(int_results2[0].name)
```

Comment:
What I meant with this suggestion was to show all the result names, so something like this:

```suggestion
            print(f"Available results: {int_results2}")
```

Reply:
It is not my joke, completely misunderstood! 
done.

Reply:
Sorry, I just realized my suggestion had an error in it. I meant to print just the names like:

```suggestion
            print(f"Available results: {[r.name for r in int_results2]}")
```

Reply:
Also, by the way, you said "done" but I don't see a new commit so far.

Reply:
Hi, @wshanks.
I apologize for my delay.